src/ProgressOnderwijsUtils/SchemaReflection/DbColumnMetaData.cs

```diff
@@ -81,20 +83,24 @@ public static DbColumnMetaData[] ColumnMetaDatas(SqlCommandCreationContext conn,
             => ColumnMetaDatas(conn, objectName.CommandText());
 
         public static DbColumnMetaData[] ColumnMetaDatas(SqlCommandCreationContext conn, string qualifiedObjectName)
-            => BaseQuery(ParameterizedSql.Empty).Append(SQL($@"
-                    and c.object_id = object_id({qualifiedObjectName})
-                order by c.column_id
-                "))
-                .ReadMetaObjects<DbColumnMetaData>(conn);
+        {
+            if (qualifiedObjectName.StartsWith("#")) {
+                return BaseQuery(tempDb).Append(SQL($@"
+                        and c.object_id = object_id({$"{tempDb.CommandText()}..{qualifiedObjectName}"})
+                    order by c.column_id
+                ")).ReadMetaObjects<DbColumnMetaData>(conn);
+            } else {
+                return BaseQuery(ParameterizedSql.Empty).Append(SQL($@"
+                        and c.object_id = object_id({qualifiedObjectName})
+                    order by c.column_id
+                ")).ReadMetaObjects<DbColumnMetaData>(conn);
+            }

```

Comment:
je zou ook een tempDbQualificationPrefix kunnen maken alla `qualifiedObjectName.StartsWith("#") ? "tempdb..":""`, en dan is de scope van deze if kleiner.